clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -308,7 +310,9 @@ function registerCommands(
             name: "Copybooks",
           });
         } catch (_error) {
-          return Error(`Can't create copybook internal folder`);
+          vscode.window.showErrorMessage(
+            `Can't create copybook internal folder`,
```

Comment:
`Failed to create an internal copybook folder` (in both places). Could the original `_error` be included to communicate the reason for the failure?

Reply:
errors are unpredictable so I added them into outputchannel : bd0468d4aee78ab7d2d06535ddb3bf7a4733f515

Reply:
I belive the suggestion here is to change the message to `Failed to create an internal copybook folder: ${error}` because tech docs prefer not to use abbreviated words like `Can't`.

Reply:
should resolve 83f21931e1b227c9067719b2eb37d9de80e20cfe